tests/test_photon_counting.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+import os
+import astropy.time as time
+import corgidrp.mocks as mocks
+import corgidrp
+from astropy.io import fits
+import numpy as np
+import corgidrp.walker as walker
+import corgidrp.caldb as caldb
+import corgidrp.data as data
+import corgidrp.detector as detector
+from corgidrp.photon_counting import get_pc_mean, photon_count, PhotonCountException
+
+def test_negative():
+    """Values at or below the 
+    threshold are photon-counted as 0, including negative values."""
+    fr = np.array([-3,2,0])
+    pc_fr = photon_count(fr, thresh=2)
+    assert (pc_fr == np.array([0,0,0])).all()
+    
+
+def test_pc():
+    '''Test that a pixel that is masked heavily (reducing the number of usable frames for that pixel) has a bigger err than the average pixel.
```

Comment:
The docstring is a bit hard to read. Can we put each thing that is being tested in a new line?

We should also mention that we test the dark subtraction from illuminated frames gives the expected value.